extensions/wikia/UserLogin/UserLoginForm.class.php

```diff
@@ -52,24 +52,30 @@ function load() {
 		$this->wpUserBirthDay = strtotime( $this->wpBirthYear . '-' . $this->wpBirthMonth . '-' . $this->wpBirthDay );
 	}
 
-	// add new account
+	/**
+	 * Adds a new user account and sends a confirmation email.
+	 *
+	 * @return bool|User an instance of User on success; boolean false otherwise.
```

Comment:
I wonder if returning `null` on failure makes the interface a bit simpler; result is still truthy.